### PR TITLE
[NLU-3742] SPIKE: Add catalan entity support

### PR DESCRIPTION
**Pull request by samhickey25**

### CODE REVIEW

File: Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_date.py [hunk 1]
```diff
@@ -235,7 +235,7 @@ def extract(self, source: str, reference: datetime = None) -> List[ExtractResult
         tokens.extend(self.basic_regex_match(source))
         tokens.extend(self.implicit_date(source))
         tokens.extend(self.number_with_month(source, reference))
-        tokens.extend(self.relative_duration_date(source, reference))
+        # tokens.extend(self.relative_duration_date(source, reference))
```
Comment (samhickey25) [inline Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_date.py:238]: this is the one thing that would need to be resolved if we were to implement this minimal support. Can't skip this extraction for other languages

Comment (colm-dillon) [reply Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_date.py:238]: So we want to skip for catalan only?

Comment (samhickey25) [reply Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_date.py:238]: yup

File: Python/tests/runner.py [hunk 2]
```diff
@@ -72,19 +72,19 @@ def get_specs(recognizer, entity):
 ENTITY_PATTERN = re.compile('(.*)(Model|Parser|Extractor|Resolver)(.*)')
 
 CULTURES = {
```
Comment (colm-dillon) [inline Python/tests/runner.py:74]: Presume I'm adding Catalan for a new test run. Do I need to add test cases for Catalan? Where should we get them from? @samhickey25

Comment (samhickey25) [reply Python/tests/runner.py:74]: Definitely need to uncomment these other languages. We don't have any test cases for Catalan. At least not ones within MS Recognisers

**Pull request closed without merging**